### DISCUSSION TITLE
Should squants differentiate between Active, Reactive and Rated Power?

### DISCUSSION THREAD

**Discussion by t-ober** [ie3-institute/simona#580] [General]
Currently we only utilize `squants.power`. I am opening the discussion to at some point discuss benefits and drawbacks of differentiating between the different "types" of power.

**Comment by sebastian-peter** [comment]
I think it should. Conversion methods would be really convenient then.

**Reply by danielfeismann** [reply]
I think so too.

**Comment by danielfeismann** [comment]
Closed by #846
